scripts/build-newsroom-videos.js

```diff
@@ -0,0 +1,39 @@
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const fetch = require('node-fetch')
+
+async function buildNewsroomVideos () {
+    
+    try{
+        let data;
+        const response = await fetch('https://youtube.googleapis.com/youtube/v3/search?' + new URLSearchParams({
+          key: process.env.YOUTUBE_TOKEN,
```

Comment:
Hey @derberg, while building this file locally, I found that `process.env.YOUTUBE_TOKEN` is not accessible from this file. Is it the same in production as well?

Reply:
As I remember, in dev mode (even in PRs) we don't propagate some envs due to security issues.

Reply:
I know that in PRs, the environment variables set in Netlify are not accessible but this statement even can't able to access environment variables from `.env` or `.env.local` file.